::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 19 Nov 2016 20:39:35 +0100
Subject: [PATCH] f_odlpluginbuild: Armband prereq in packages.yaml

We need to explicitily add our Armband leveldb patching prereq to
the ODL plugin's packages.yaml, so they are included in the
offline Ubuntu/MOS mirrors.

After ARMBAND-114 was implemented, the dependency list is only:
- armband-odl-leveldb-fix (for patching leveldbjni);
- libasound2 (openjdk-8-jre amos version additional dep);

Note: armband-odl-leveldb-fix MUST be also added to MOS mirror
package list, since it's not present in Ubuntu repos.

JIRA: ARMBAND-167
JIRA: ARMBAND-114

FIXME: These are only required for arm64, so maybe later we could
filter out other archs by extending the packages.yaml with an <arch>
field.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/f_isoroot/f_odlpluginbuild/packages.yaml | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 build/f_isoroot/f_odlpluginbuild/packages.yaml

diff --git a/build/f_isoroot/f_odlpluginbuild/packages.yaml b/build/f_isoroot/f_odlpluginbuild/packages.yaml
new file mode 100644
index 0000000..b1bf086
--- /dev/null
+++ b/build/f_isoroot/f_odlpluginbuild/packages.yaml
@@ -0,0 +1,5 @@
+# Armband: openjdk-8-jre depends on libasound2
+# Armband: leveldb patching requires armband-odl-leveldb-fix
+packages:
+  - armband-odl-leveldb-fix
+  - libasound2
